sample-collector-config.yaml

```diff
@@ -65,44 +65,6 @@ processors:
               - source_attributes: ["http.url"]
                 span_kinds: ["Client", "Server"]
                 regex: "https?://.+?((/[^/?]+){1,2}).*"
-    custom_attributes:
```

Comment:
Don't we want the equivalent config in the new format?

Reply:
Yes, the `span_attributes` changes for the sample config were already checked-in before